sensor/stmemsc/_prj_MKI109V3/_prj_MKI109V3.ioc

```diff
@@ -0,0 +1,304 @@
+#MicroXplorer Configuration settings - do not modify
```

Comment:
I wonder if we should include these .ioc files since they are not used.
I think this might cause confusion.
Besides this is STM32 specific, which is not the purpose of this module.

Reply:
OK, I'll remove it.